run_in_docker.sh

```diff
@@ -35,9 +35,9 @@ if [[ ! -e $HOME/.config/gcloud/application_default_credentials.json ]]; then
 fi
 
 docker run \
-    -v $HOME/.config/gcloud:/root/.config/gcloud \
-    -v ${context}:/workspace \
+    -v "$HOME"/.config/gcloud:/root/.config/gcloud \
+    -v "$context":/workspace \
     gcr.io/kaniko-project/executor:latest \
-    --dockerfile ${dockerfile} --destination ${destination} --context dir:///workspace/ \
-    --cache=${cache}
+    --dockerfile "${dockerfile}" --destination "${destination}" --context "${context}" \
```

Comment:
Since we mount $context to `/workspace` in the container itself, we want to  point kaniko to `/workspace`